scripts/spidhub-fargate/environments/uat/storage/config/spid-saml-check/v1/configvolume/spid_users.json

```diff
@@ -0,0 +1,562 @@
+		"mobilePhone": "3939393939",
+		"email": "aabyron@hotmail.com",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "garibaldi",
+		"password": "password123",
+		"spidLevel": 1,
+
+		"spidCode": "SPID-003",
+		"familyName": "Garibaldi",
+		"name": "Giuseppe Maria",
+		"gender": "M",
+		"dateOfBirth": "1987-07-04",
+		"countyOfBirth": "CE",
+		"placeOfBirth": "L741",
+		"idCard": "patenteGuida U1U111111U motorizzazioneCaserta 2013-04-03 2023-07-04",
+		"expirationDate": "2019-03-13",
+		"fiscalNumber": "TINIT-GRBGPP87L04L741X",
+		"mobilePhone": "3310001000",
+		"email": "eroedei2mondi@yahoo.it",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "lucrezia",
+		"password": "password123",
+		"spidLevel": 1,
+
+		"spidCode": "SPID-004",
+		"familyName": "Borgia",
+		"name": "Lucrezia",
+		"gender": "F",
+		"dateOfBirth": "1980-04-18",
+		"countyOfBirth": "RM",
+		"placeOfBirth": "H501",
+		"idCard": "cartaIdentita AK987654 comuneFerrara 2012-11-28 2023-04-18",
+		"expirationDate": "2019-01-01",
+		"fiscalNumber": "TINIT-BRGLRZ80D58H501Q",
+		"mobilePhone": "3495555555",
+		"email": "luborgia@msn.com",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "cristoforocolombo",
+		"password": "password123",
+		"spidLevel": 2,
+
+		"spidCode": "SPID-005",
+		"familyName": "Colombo",
+		"name": "Cristoforo",
+		"gender": "M",
+		"dateOfBirth": "1942-10-12",
+		"countyOfBirth": "GE",
+		"placeOfBirth": "D969",
+		"idCard": "patenteNautica 102579 capitaneriaPortoGenova 2014-08-05 2024-10-12",
+		"expirationDate": "2018-09-15",
+		"fiscalNumber": "TINIT-CLMCST42R12D969Z",
+		"mobilePhone": "3893893893",
+		"email": "laninalapintaelasantamaria@outlook.com",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "lapulzella",
+		"password": "password123",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)